lib/plugins/aws/invoke.js

```diff
@@ -113,6 +113,14 @@ class AwsInvoke {
       );
       const logResult = Buffer.from(invocationReply.LogResult, 'base64').toString();
       logResult.split('\n').forEach((line) => {
+        if (
+          // TODO: Switch to .startsWith after otel extension is improved
+          line.includes('⚡.') ||
```

Comment:
It's long time since we do not issue such logs with extension, so I think it can safely be removed